### ISSUE TITLE
Vedastr communication group.

### ISSUE THREAD

**Issue by ChaseMonsterAway** [Media-Smart/vedastr#44]
We have a communication group now. If you have some problems or want to communicate with other OCRer, you can join this group.
![微信图片_20210612214804](https://user-images.githubusercontent.com/58807745/121778153-45cea380-cbc8-11eb-9253-6edb10187805.jpg)

**Comment by jdavidd** [comment]
Hello, I want to join this group, but after I install the app they show me a QR code  and say that someone with weChat app to scan it, could you help me with that?

**Comment by ChaseMonsterAway** [comment]
> Hello, I want to join this group, but after I install the app they show me a QR code and say that someone with weChat app to scan it, could you help me with that?

You should install the wechat app in your mobile phone first.

**Comment by jdavidd** [comment]
Yes it's installed, but now I need someone from China, to scan my QRCode.
![126070968_1028507881002250_5411695013782781874_n](https://user-images.githubusercontent.com/26500800/99522630-8ea9db00-299e-11eb-84ef-6fa3612ef3b9.jpg)

But I think, the qrCode generate a code only for 5 minute, I dont know exactly.

**Comment by ChaseMonsterAway** [comment]
> Yes it's installed, but now I need someone from China, to scan my QRCode.
> ![126070968_1028507881002250_5411695013782781874_n](https://user-images.githubusercontent.com/26500800/99522630-8ea9db00-299e-11eb-84ef-6fa3612ef3b9.jpg)
> 
> But I think, the qrCode generate a code only for 5 minute, I dont know exactly.

Yes, It is invalid now.

**Comment by jdavidd** [comment]
Do you have an email? Or an hour, to be live =))) so I can send the QRCode?

**Comment by ChaseMonsterAway** [comment]
> Do you have an email? Or an hour, to be live =))) so I can send the QRCode?

You can paste it here now. I will keep online for about 30 minutes.

**Comment by jdavidd** [comment]
![f](https://user-images.githubusercontent.com/26500800/99534558-4398c380-29b0-11eb-8479-58de9accdb5c.jpg)
here, i think its available 5 minutes

**Comment by ChaseMonsterAway** [comment]
> ![f](https://user-images.githubusercontent.com/26500800/99534558-4398c380-29b0-11eb-8479-58de9accdb5c.jpg)
> here, i think its available 5 minutes

No. invalid now. I scan it when it in 4 minutes.

**Comment by jdavidd** [comment]
![vf](https://user-images.githubusercontent.com/26500800/99535265-56f85e80-29b1-11eb-8ccb-4614c2ac28ce.jpg)
Ok, they say this is an other method, you should type my number phone.

**Comment by ChaseMonsterAway** [comment]
> ![vf](https://user-images.githubusercontent.com/26500800/99535265-56f85e80-29b1-11eb-8ccb-4614c2ac28ce.jpg)
> Ok, they say this is an other method, you should type my number phone.

It's okay now.

**Comment by jdavidd** [comment]
I did it, thanks :D

**Comment by yufish** [comment]
又过期了， 能再发一个么？

**Comment by yufish** [comment]
多谢~~

**Comment by chibohe** [comment]
Could you please repost a QR code, thx.

**Comment by ChaseMonsterAway** [comment]
> Could you please repost a QR code, thx.

I have updated it.

**Comment by rexlow** [comment]
The QR code is outdated. Can you please refresh it? @ChaseMonsterAway

**Comment by ChaseMonsterAway** [comment]
> The QR code is outdated. Can you please refresh it? @ChaseMonsterAway

I have updated it.

**Comment by Johnson-yue** [comment]
The QR code is outdated @ChaseMonsterAway  Can you please refresh it?

**Comment by ChaseMonsterAway** [comment]
> The QR code is outdated @ChaseMonsterAway Can you please refresh it?

I've refreshed it.

**Comment by Johnson-yue** [comment]
@ChaseMonsterAway thanks

**Comment by w867066886** [comment]
The QR code is outdated. Can you please refresh it? @ChaseMonsterAway

**Comment by xxAna** [comment]
@ChaseMonsterAway,Hi,Can you please refresh the QR code? Best wishes!

**Comment by ChaseMonsterAway** [comment]
@w867066886 @xxAna 
Hi, I have updated the QR code. Sorry for late reply.

**Comment by YuNaruto** [comment]
The QR code is outdated, Can you please refresh it? thx.
@ChaseMonsterAway

**Comment by Amoko** [comment]
Could you please update the QR code, if it's alright. thx @ChaseMonsterAway

**Comment by hlzzzzzzz** [comment]
QR码已过时，可以刷新一下吗？感谢。
@ChaseMonsterAway

**Comment by Y-pandaman** [comment]
二维码已经过了，可以更新一下吗？谢谢。
